cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1899,6 +1905,39 @@ pub(crate) fn define(
         e.enc_32_64_maybe_isap(instruction, template, None); // from SSE
     }
 
+    // SIMD register movement: store, load, spill, fill, regmove. All of these use variants of
+    // MOVUPS from SSE (TODO ideally this would use the aligned MOVAPS).
```

Comment:
Does a perf penalty occur if we use MOVUPS instead of e.g. MOVUPD for float64x2, or MOVDQU for integer SIMD types?

Reply:
Just a cursory glance seems to indicate that, yes, there seem to be faster encodings for different types. For this PR I was still just trying to get the wasmtime spec tests to work so I was not thinking too much about performance yet. Let me open an issue for this one because this is exactly the kind of thing I will be spending more time on after this is all working: https://github.com/CraneStation/cranelift/issues/1039.